addons/http_routing/models/ir_http.py

```diff
@@ -297,8 +297,10 @@ def get_frontend_session_info(self):
         lang = user_context.get('lang')
         translation_hash = request.env['ir.translation'].get_web_translations_hash(modules, lang)
 
+        lang_code = request.lang._get_cached('url_code')
+        lang_prefix = ('/' + lang_code) if lang_code != 'en' else ''
```

Comment:
Didn't really checked (yet) the PR yet but saw it passing by:
1. url_code is a configurable field, checking `'en'` is not correct, might be `english` or `en_US` or whatever is some DBs 
2. Why targeting the english lang? Some DB can be initialized with another lang only

Also, might be worth a comment `# avoid 303 redirect in case we are on translated version`

Reply:
It's cleaner now and I just use existing function `url_lang`